lib/nostrum/shard/event.ex

```diff
@@ -32,10 +32,7 @@ defmodule Nostrum.Shard.Event do
 
   def handle(:heartbeat_ack, _payload, state) do
     Logger.debug "HEARTBEAT_ACK"
-    heartbeat_intervals = state.heartbeat_intervals
-    |> List.delete_at(-1)
-    |> List.insert_at(0, Util.now() - state.last_heartbeat)
-    %{state | heartbeat_intervals: heartbeat_intervals}
```

Comment:
In case of discord outage, the gateway generally keep sending heartbeat_ack events, even if nothing is working. So I don't really know about the relevance of tracking that.

```
It is worth noting that in the event of a service outage where you stay connected to the gateway, you should continue to heartbeat and receive ACKs. The gateway will eventually respond and issue a session once it is able to.
```
From https://discordapp.com/developers/docs/topics/gateway#gateway-heartbeat